pages/design/rule-template.md

```diff
@@ -4,16 +4,16 @@ title: Auto-WCAG Rule Template
 
 The rule template contains a plain language description of the rule, some background information, and a list of all assumptions. The test procedure is defined by the selector, a number of steps and a description of the possible outcomes.
 
-Use the [empty test template](rule-template-empty.html) to create new auto-wcag rule. When creating a new rule, first read [rule design](rule-design.html).
+Use the [empty atomic rule template](atomic-template-empty.html) to create new auto-wcag rule. When creating a new rule, first read [rule design](rule-design.html).
 
-```markdown
+````markdown
 ---
 name: Short descriptive name
-group: group-id # Optional
+rule_type: atomic
```

Comment:
Can we align the two templates, so that people get the same level of guidance no matter where they find the template? And like me, don't get confused about which is which - I think I wrote some comments for the wrong ones now... 
Aligning the two would also make it easier to keep them aligned.